js/main.js

```diff
@@ -1,141 +0,0 @@
-// API key set globally so that it is easily edited if needed
-const apiKey = '7e8e7a9139e4f09fb253db0d8ddce620';
-
-// Zip value set globally to be edited later by user input
-let zip = ''
-
-
-// Values set globally to be edited later with stateValue()
-let weatherConditions = [
-    {
-        city: '',
-        kTemp: '',
-        fahrenheitTemp: '',
-        celsiusTemp: '',
-        condition: '',
-        icon: '',
-
-    }
-]
-
-// Gets values from API
-// If zip causes API to return an error
-// catch - alert message will run
-async function getWeather() {
```

Comment:
You could set this up with a parameter in case you add geolocation or other options in the request. 

Axios allows you to do `axios.get(url, options)` where the options become `{ params: { zip, appid }}`. You could do that instead of getting the zip from a global variable. 

Additionally, `https://api.openweathermap.org/data/2.5/weather` doesn't change so you could set that to a variable like `API_KEY` would be

Here you could pass this object as a second param to `.get` to make it better written.
`{ params: { zip, appid: APIKey } }`